components/common-alt/search/index.jsx

```diff
@@ -7,14 +7,16 @@ import { removeFileExtension } from "@utils/files"
 import Link from "next/link"
 
 export const SearchResult = ({ result, selected, onMouseOver, onClick }) => {
+  const url = result.path.replace("\\index","");
```

Comment:
This is essential because the search results from Fuse.js results the path /index for folders whereas the folder pages loads up without the index in the URL. This sorts it out.